src/Commands/ClearNotifications.php

```
@@ -0,0 +1,43 @@
+#[AsCommand(name: 'notifications:clear')]
+final class ClearNotifications extends Command
+{
+    #[Override]
+    protected function configure(): void
+    {
+        $this->setDescription('Clear all notifications, past and future.')
+            ->setHelp('Will remove all notifications from the notifications mysql table.');
+    }
+
+    #[Override]
+    protected function execute(InputInterface $input, OutputInterface $output)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$input'. (undefined)

(UnusedFormalParameter)